### ISSUE TITLE
Error when compiling v12.8.1 on posix 

### ISSUE THREAD

**Issue by abaga129** [AuburnSounds/Dplug#720]
I'm receiving the following error.

```
../../.dub/packages/dplug-12.8.1/dplug/window/dplug/window/win32window.d(1154,29): Error: undefined identifier `LRESULT`
../../.dub/packages/dplug-12.8.1/dplug/window/dplug/window/win32window.d(1154,29): Error: undefined identifier `WPARAM`
../../.dub/packages/dplug-12.8.1/dplug/window/dplug/window/win32window.d(1154,29): Error: undefined identifier `LPARAM`
```

Looks like the new function `LowLevelMouseProc` just needs to be wrapped in `version (Windows)` I'll open a PR :)

**Comment by abaga129** [comment]
I just noticed that my auto-formatter made several changes.  Let me know if this is not okay and I'll happily fix it!

**Comment by p0nce** [comment]
This is fixed in v12.8.2, my bad
